web_src/js/features/repo-findfile.js

```diff
@@ -72,6 +72,10 @@ export function filterRepoFilesWeighted(files, filter) {
   return filterResult;
 }
 
+export function escapePath(s) {
+  return s.split('/').map(encodeURIComponent).join('/');
+}
```

Comment:
Move function to `utils.js`? I think such simple pure functions belong there.

Reply:
I have thought about it, but I didn't do it for 2 reasons:

1.  Now the `escapePath` is only used by `repo-findfile.js`, not sure whether it is general enough for frontend.
2. The  `utils.js` have already contained many different functions, maybe in the future it should be refactored to `utils/color.js`, `utils/string.js`, `utils/html.js`, etc. I do not feel it's good to add too many functions to it now.


So, I prefer to keep the escapePath here. If it's really needed to be used somewhere else, let's do refactor.